docs/source/status.rst

```diff
@@ -0,0 +1,199 @@
+| gRPC transport              | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| gRPC + TLS transport        | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| RPC error codes             | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| Authentication handlers     | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| Custom client middleware    | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| Custom server middleware    | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+
+.. seealso::
+   The :ref:`flight-rpc` specification.
+
+
+C Data Interface
+================
+
++-----------------------------+-------+--------+-------+
+| Feature                     | C++   | Python | R     |
+|                             |       |        |       |
++=============================+=======+========+=======+
+| Schema export               | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+| Array export                | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+| Schema import               | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+| Array import                | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+
+.. seealso::
+   The :ref:`C Data Interface <c-data-interface>` specification.
+
+
+Third-Party Data Formats
+========================
+
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| Format                      | C++     | Java    | Go    | JavaScript | C#    | Rust  |
+|                             |         |         |       |            |       |       |
++=============================+=========+=========+=======+============+=======+=======+
+| Avro                        |         | R       |       |            |       |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| CSV                         | R       |         | R/W   |            |       | R/W   |
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| ORC                         | R       |         |       |            |       |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| Parquet                     | R/W     |         |       |            |       |       |
```

Comment:
Rust also has a parquet implementation AFAIK

Reply:
I didn't fill it in intentionally on the assumption that it's Arrow <> Parquet support, which is still incomplete. We have read support, but not yet of nested types, there might be other limitations on the read side. We've only recently started with write support, so that might take some time.

I'm planning on updating the Rust support before 1.0.0 based on what integration tests we pass or fail. I can also update the Parquet support then.